site/content/blog/2022-08-22-the-param-miner-addon/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "The Param Miner Add-on"
+summary: >
+  The parameter discovery add-on for ZAP.
+images:
+- https://www.zaproxy.org/blog/2022-08-22-the-param-miner-addon/images/paramminer.png
+type: post
+tags:
+- blog
+date: "2022-08-22"
+authors: 
+    - Arkaprabha
+---
+
+Parameter discovery is the process of finding unlinked, unreferenced, or hidden parameters that can be used by the user to control or influence the behaviour of the application. Resulting in successful attack scenarios like SQL injection, XSS, SSRF, Web Cache Poisoning based attacks and much more. Often times such uncovered parameters are properly configured and can be a suitable entry point for further manipulation. These are real hidden treasures which can be very crucial in the recon process to develop a successful exploit.  
+
+Param Miner is a new add-on for ZAP that allows you to discover and analyse parameters in web applications, that might not be visible to the user. The main idea behind Param-Miner is to increase the attack surface of the website by identifying these parameters.
```

Comment:
First "Param Miner" (Digger;) mention could link to the help pages, which need to be put on the Marketplace...